### PR TITLE
chore: update select component copy

### PR DESCRIPTION
**Pull request by rossmoody**
## Description
This PR introduces updated copy from UXCS from the `Dropdowns/Select` component page. 

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/pages/components/select.mdx [hunk 1]
```diff
@@ -22,12 +22,12 @@ components:
...
-        To filter a large list of options, consider an{' '}
-        <a href="/components/autocomplete">Autocomplete</a>
+        To filter a large list of options, use <a href="/components/autocomplete">Autocomplete</a>{' '}
+        instead
       </li>
       <li>
-        To make multiple selections from a list, use a{' '}
-        <a href="/components/multiselect">Multiselect</a>
+        To make multiple selections from a list, use{' '}
+        <a href="/components/multiselect">Multiselect</a> instead
```
Comment (jzempel) [inline src/pages/components/select.mdx:30]: Just a reminder comment... Use `<Markdown>` if you prefer to use `[link](url)` style syntax for links.

File: src/pages/components/select.mdx [hunk 2]
```diff
@@ -22,12 +22,12 @@ components:
   <Misuse>
     <ul>
       <li>
-        To filter a large list of options, consider an{' '}
-        <a href="/components/autocomplete">Autocomplete</a>
+        To filter a large list of options, use
+        <Markdown>[Autocomplete](/components/autocomplete)</Markdown> instead
```
Comment (allisonacs) [inline src/pages/components/select.mdx:26]: Missing a space between words:
image

File: src/pages/components/select.mdx [hunk 3]
```diff
@@ -22,12 +22,12 @@ components:
...
-        To filter a large list of options, consider an{' '}
-        <a href="/components/autocomplete">Autocomplete</a>
+        To filter a large list of options, use{' '}
+        <Markdown>[Autocomplete](/components/autocomplete)</Markdown> instead
       </li>
       <li>
-        To make multiple selections from a list, use a{' '}
-        <a href="/components/multiselect">Multiselect</a>
+        To make multiple selections from a list, use{' '}
+        <Markdown>[Multiselect](/components/multiselect)</Markdown> instead
```
Comment (jzempel) [inline src/pages/components/select.mdx:30]: @rossmoody the expected JSX formatting would be to do something similar to:

```jsx
<li><Markdown>
  words go here
</Markdown></li>
```

In this way, you set up the entire block to contain whatever markdown is needed either now or in the future.

Comment (rossmoody) [reply src/pages/components/select.mdx]: Wonderful, ty.

**Pull request merged**